### DISCUSSION TITLE
Make elements in playfield layer always visible

### DISCUSSION THREAD

**Discussion by Loluh** [ppy/osu#26806] [Ideas]
I want to be able to see some skin elements while rest of HUD is hidden while playing, so i'm suggesting that elements in playfield layer are visible even when HUD is hidden during gameplay,

Currently on stable it works with UR bar and key presses:
![image](https://github.com/ppy/osu/assets/87331677/c7d2f5fc-f6f0-45a7-a83f-c5d486d7a150)

I would like it to work on lazer like that, I'm aware that it has an option for key presses, but i want to have UR and other different elements aswell.

**Comment by Gabixel** [comment]
Probably in connection with this suggestion:
- https://github.com/ppy/osu/issues/12039
- https://github.com/ppy/osu/discussions/23323
- https://github.com/ppy/osu/discussions/24193
- https://github.com/ppy/osu/discussions/23236
